src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
```

Comment:
because `onServerResponse` is an arrow method and therefore bound to this, you can just pass it directly as a parameter.